event_routing_backends/settings/common.py

```diff
@@ -59,84 +60,119 @@ def plugin_settings(settings):
+            'edx.forum.comment.deleted',
+            'edx.forum.comment.edited',
+            'edx.forum.comment.reported',
+            'edx.forum.comment.unreported',
+            'edx.ui.lms.link_clicked',
+            'edx.ui.lms.sequence.outline.selected',
+            'edx.ui.lms.outline.selected',
+            'edx.ui.lms.sequence.next_selected',
+            'edx.ui.lms.sequence.previous_selected',
+            'edx.ui.lms.sequence.tab_selected',
+            'showanswer',
+            'edx.problem.hint.demandhint_displayed',
+            'problem_check',
+            'load_video',
+            'edx.video.loaded',
+            'play_video',
+            'edx.video.played',
+            'complete_video',
+            'edx.video.completed',
+            'stop_video',
+            'edx.video.stopped',
+            'pause_video',
+            'edx.video.paused',
+            'seek_video',
+            'edx.video.position.changed',
+            'hide_transcript',
+            'edx.video.transcript.hidden',
+            'show_transcript',
+            'edx.video.transcript.shown',
+            'speed_change_video',
+            'video_hide_cc_menu',
+            'edx.video.closed_captions.shown',
+            'edx.video.closed_captions.hidden',
+            'edx.video.language_menu.hidden',
+            'video_show_cc_menu',
+            'edx.video.language_menu.shown',
+            'edx.course.grade.passed.first_time'
+        ],
+        'caliper': [
+            'edx.course.enrollment.activated',
+            'edx.course.enrollment.deactivated',
+            'edx.ui.lms.link_clicked',
+            'edx.ui.lms.sequence.outline.selected',
+            'edx.ui.lms.outline.selected',
+            'edx.ui.lms.sequence.next_selected',
+            'edx.ui.lms.sequence.previous_selected',
+            'edx.ui.lms.sequence.tab_selected',
+            'showanswer',
+            'edx.problem.hint.demandhint_displayed',
+            'problem_check',
+            'load_video',
+            'edx.video.loaded',
+            'play_video',
+            'edx.video.played',
+            'complete_video',
+            'edx.video.completed',
+            'stop_video',
+            'edx.video.stopped',
+            'pause_video',
+            'edx.video.paused',
+            'seek_video',
+            'edx.video.position.changed',
+            'edx.course.grade.passed.first_time',
+            'edx.course.grade.now_passed',
+            'edx.course.grade.now_failed'
+        ],
+    }
+
+    merged_whitelist = set(settings.WHITELIST_EVENTS['xapi']).union(set(settings.WHITELIST_EVENTS['caliper']))
+
     settings.EVENT_TRACKING_BACKENDS.update({
-        'xapi': {
-            'ENGINE': 'eventtracking.backends.async_routing.AsyncRoutingBackend',
-            'OPTIONS': {
-                'backend_name': 'xapi',
-                'processors': [
+        "event_bus": {
+            "ENGINE": "eventtracking.backends.event_bus.EventBusRoutingBackend",
```

Comment:
We should not change the default behavior, the event bus is experimental through at least Quince.

Reply:
Does this must go to the tutor-contrib-aspects plugin behind a setting?

Reply:
I think Aspects should not care much about how ERB is configured, aside from adding Ralph as a backend when appropriate. Since ERB is installed by default, we don't want to mess with any existing settings and should probably just document the settings in ERB and point to them from Aspects. I don't want Aspects to fight users' settings in upstream modules.

Reply:
I think this comment is still relevant, and the same for changes below?